src/content/en/fundamentals/building-components/best-practices.md

```diff
@@ -0,0 +1,535 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A checklist which breaks down all the things we think it takes to be a well behaved Custom Element.
```

Comment:
quick question. What about Shadow DOM? Should that be 'shadow DOM' throughout as well?